xml/System.Diagnostics/ActivitySource.xml

```diff
@@ -63,8 +63,8 @@
         <Parameter Name="listener" Type="System.Diagnostics.ActivityListener" />
       </Parameters>
       <Docs>
-        <param name="listener">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="listener">The activity listener object to use for listening to the activity events.</param>
+        <summary>Adds a listener to the activity starting and stopping events.</summary>
```

Comment:
>Adds a listener to the activity starting and stopping events. [](start = 17, length = 61)

Adds a listener to the activity starting and stopping events for the Activities created by the current ActivitySource..